packages/components/src/components/table/table-head-cell/table-head-cell.tsx

```diff
@@ -54,10 +60,23 @@ export class TableHeadCell {
     return (
       <Host scope="col" role="columnheader" aria-sort={getAriaSort(this.sort)}>
         {isSortable(active, direction) ? (
-          <button type="button" onClick={this.onButtonClick}>
-            <slot />
-            <PrefixedTagNames.pIcon class="icon" color="inherit" name="arrow-up" aria-hidden="true" />
-          </button>
+          <PrefixedTagNames.pButtonPure
```

Comment:
We have to use the implementation like it was done before. Because with the new implementation we waste a lot of Dom Nodes and can't achieve SSR theme support.

Reply:
ok, I'll change it

Reply:
Done